src/components/base/Button.js

```diff
@@ -46,6 +46,7 @@ export default function Button(props) {
       className={props.className}
       type={props.type}
       aria-label={props.children}
+      data-testid={props.dataTestid}
```

Comment:
Pour les composants comme ca c'est bien de faire passe plat => on est pas censé connaitre l'implem de `Button`
```suggestion
      data-testid={props['data-testid']}
```

Reply:
Aïe je n'ai pas compris, à discuter sur mattermost